### PR TITLE
Update start-a-local-cluster-in-docker-windows.md

### PR DESCRIPTION
**Pull request by JayStag**
@Amruta-Ranade @jseldess Update Step 6 with correct per issue #7207 PS command line to remove nodes' data stores.

### CODE REVIEW

File: v19.2/start-a-local-cluster-in-docker-windows.md [hunk 1]
```diff
@@ -26,171 +26,179 @@ Running multiple nodes on a single host is useful for testing CockroachDB, but i
...
-    --name=roach1 `
-    --hostname=roach1 `
-    --net=roachnet `
-    -p 26257:26257 -p 8080:8080  `
-    -v "//c/Users/<username>/cockroach-data/roach1:/cockroach/cockroach-data"  `
-    {{page.release_info.docker_image}}:{{page.release_info.version}} start `
-    --insecure `
-    --join=roach1,roach2,roach3
-    ~~~
+   ```powershell
```
Comment (Amruta-Ranade) [inline v19.2/start-a-local-cluster-in-docker-windows.md:41]: I'm not sure why we need to change ~~~ to ``` in the doc. Mind sharing your reasoning?

File: v19.2/start-a-local-cluster-in-docker-windows.md [hunk 2]
```diff
@@ -26,171 +26,179 @@ Running multiple nodes on a single host is useful for testing CockroachDB, but i
...
-    - `-v "//c/Users/<username>/cockroach-data/roach1:/cockroach/cockroach-data"`: This flag mounts a host directory as a data volume. This means that data and logs for this node will be stored in `Users/<username>/cockroach-data/roach1` on the host and will persist after the container is stopped or deleted. For more details, see Docker's <a href="https://docs.docker.com/engine/admin/volumes/bind-mounts/">Bind Mounts</a> topic.
-    - `{{page.release_info.docker_image}}:{{page.release_info.version}} start --insecure --join`: The CockroachDB command to [start a node](cockroach-start.html) in the container in insecure mode. The `--join` flag specifies the `hostname` of each node that will initially comprise your cluster. Otherwise, all [`cockroach start`](cockroach-start.html) defaults are accepted. Note that since each node is in a unique container, using identical default ports won’t cause conflicts.
+
```
Comment (Amruta-Ranade) [inline v19.2/start-a-local-cluster-in-docker-windows.md:54]: This is an unnecessary formatting change that breaks the steps numbering. Please revert the change.

File: v19.2/start-a-local-cluster-in-docker-windows.md [hunk 3]
```diff
@@ -26,171 +26,179 @@ Running multiple nodes on a single host is useful for testing CockroachDB, but i
...
+   - `-v "//c/Users/<username>/cockroach-data/roach1:/cockroach/cockroach-data"`: This flag mounts a host directory as a data volume. This means that data and logs for this node will be stored in `Users/<username>/cockroach-data/roach1` on the host and will persist after the container is stopped or deleted. For more details, see Docker's <a href="https://docs.docker.com/engine/admin/volumes/bind-mounts/">Bind Mounts</a> topic.
+   - `{{page.release_info.docker_image}}:{{page.release_info.version}} start --insecure --join`: The CockroachDB command to [start a node](cockroach-start.html) in the container in insecure mode. The `--join` flag specifies the `hostname` of each node that will initially comprise your cluster. Otherwise, all [`cockroach start`](cockroach-start.html) defaults are accepted. Note that since each node is in a unique container, using identical default ports won’t cause conflicts.
 
 3. Start two more nodes:
 
-    {{site.data.alerts.callout_info}}Again, be sure to replace <code>&#60;username&#62;</code> in the <code>-v</code> flag with your actual username.{{site.data.alerts.end}}
```
Comment (Amruta-Ranade) [inline v19.2/start-a-local-cluster-in-docker-windows.md:65]: This is an unnecessary formatting change that breaks the steps numbering. Please revert the change.

File: v19.2/start-a-local-cluster-in-docker-windows.md [hunk 4]
```diff
@@ -26,171 +26,179 @@ Running multiple nodes on a single host is useful for testing CockroachDB, but i
...
 4. Perform a one-time initialization of the cluster:
 
-    ~~~ powershell
-    PS C:\Users\username> docker exec -it roach1 ./cockroach init --insecure
-    ~~~
+   ```powershell
+   PS C:\Users\username> docker exec -it roach1 ./cockroach init --insecure
+   ```
 
-    You'll see the following message:
```
Comment (Amruta-Ranade) [inline v19.2/start-a-local-cluster-in-docker-windows.md:97]: This is an unnecessary formatting change that breaks the steps numbering. Please revert the change.

File: v19.2/start-a-local-cluster-in-docker-windows.md [hunk 5]
```diff
@@ -26,171 +26,179 @@ Running multiple nodes on a single host is useful for testing CockroachDB, but i
...
-    (1 row)
-    ~~~
+   ```
+     id | balance
+   +----+---------+
+      1 | 1000.50
+   (1 row)
+   ```
 
-    As you can see, node 1 and node 2 behaved identically as SQL gateways.
```
Comment (Amruta-Ranade) [inline v19.2/start-a-local-cluster-in-docker-windows.md:167]: This is an unnecessary formatting change that breaks the steps numbering. Please revert the change.

**Review by Amruta-Ranade — CHANGES_REQUESTED** [review]
Please revert the formatting changes -- they are breaking the numbered lists formatting. 

The only change required for this issue is the last one (`Remove-Item C:\Users\username> cockroach-data -recurse` > `PS C:\Users\username> Remove-Item cockroach-data -recurse`). Please update the 20.1 and 20.2 docs for this change.

File: v19.2/start-a-local-cluster-in-docker-windows.md (hunk 1 above)
Comment (JayStag) [reply v19.2/start-a-local-cluster-in-docker-windows.md:41]: The only changes I deliberately made were lines 233-241 (add/remove). 🙂 If other lines changed, it was not intentional and could be my lack of experience here. I am more than happy to fix/revert as needed. 👍

Comment (Amruta-Ranade) [reply v19.2/start-a-local-cluster-in-docker-windows.md:41]: Oh..no problem! Yes, please revert the changes except the one with the wrong command. Thanks!

Comment (JayStag) [reply v19.2/start-a-local-cluster-in-docker-windows.md:41]: Will do. These changes are needed with 20.1 and 20.2 as well, yes?

Comment (JayStag) [reply v19.2/start-a-local-cluster-in-docker-windows.md:41]: 🤦🏻‍♂️ It was some settings on my code editor...

Comment (Amruta-Ranade) [reply v19.2/start-a-local-cluster-in-docker-windows.md:41]: ahh..I see. Yes, the 20.1 and 20.2 docs need to be updated as well.

Comment (JayStag) [reply v19.2/start-a-local-cluster-in-docker-windows.md:41]: 👍 Should be fixed here in a few.

**Comment by Amruta-Ranade** [comment]
hmm..the formatting's still broken. It isn't evident in the markdown file but you can check in the rendered version: http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/b9cec7665f76759c16f30aa815099960ac80c582/v19.2/start-a-local-cluster-in-docker-windows.html

**Comment by JayStag** [comment]
Hmm... Okay, this might be cleaner if I start from scratch.
